SentinelOne/sentinelone_module/logs/connector.py

```diff
@@ -0,0 +1,123 @@
+from functools import cached_property
+import time
+
+from datetime import datetime, timedelta, timezone
+from prometheus_client import Counter, Histogram
+from management.mgmtsdk_v2.exceptions import UnauthorizedException
+from management.mgmtsdk_v2.services.activity import ActivitiesFilter
+from management.mgmtsdk_v2.services.threat import ThreatQueryFilter
+from management.mgmtsdk_v2_1.mgmt import Management
+from sekoia_automation.connector import Connector
+
+from sentinelone_module.logs.configuration import SentinelOneLogsConnectorConfiguration
+
+SENTINEL_ONE_HOW_MANY_DAYS_OF_EXISTING_LOGS_TO_RETRIEVE = 1
+
+# Declare prometheus metrics
+prom_namespace = "sicconfapi_intakes"
```

Comment:
For automation modules, the prefix is `symphony_module_common`
```suggestion
prom_namespace = "symphony_module_common"
```